pkgs/applications/window-managers/sway/default.nix

```diff
@@ -1,13 +1,13 @@
 { lib, stdenv, fetchFromGitHub, substituteAll, swaybg
 , meson, ninja, pkg-config, wayland-scanner, scdoc
-, wayland, libxkbcommon, pcre, json_c, libevdev
+, wayland, libxkbcommon, pcre2, json_c, libevdev
 , pango, cairo, libinput, libcap, pam, gdk-pixbuf, librsvg
-, wlroots, wayland-protocols, libdrm
+, wlroots_0_16, wayland-protocols, libdrm
```

Comment:
I find it more future proof to override the callPackage call with `wlroots = wlroots_0_16` instead of hardcoding it here